spec/models/post_spec.rb

```diff
@@ -48,4 +48,12 @@
       expect(Post.find(post.id).tags_array).to match_array %w[wic]
     end
   end
+
+  context "unstick_at behavior" do
+    it "defaults unstick_at to 2 weeks from now when sticky is true and unstick_at is nil" do
+      post = build(:sticky_post, unstick_at: nil)
+      post.valid?
```

Comment:
```suggestion
      expect(post).to be_valid
```

Reply:
I can see that you were only calling this line to trigger validations.
An alternative solution would be to use `create` instead of `build` above, which inserts the mocked record into the (testing) DB. And DB inserts always inherently trigger validations.